### PR TITLE
The ASDKgram example doesn't compile.

### PR DESCRIPTION
**Pull request by onato**
It also crashes right away because it expects the Webservice.load method to return on the main thread.

I fixed these issues and an error whereby all valid responses return as failed.

### CODE REVIEW

File: examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -428,7 +428,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (nguyenhuy) [inline examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj]: The changes in this file are unintentional I guess?

Comment (onato) [reply examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj]: No this was intentional. It doesn't compile otherwise.

**Comment by onato** [comment]
OK, I've changed the spacing to tabs.

File: examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nguyenhuy) [reply examples_extra/ASDKgram-Swift/ASDKgram-Swift.xcodeproj/project.pbxproj]: 👍

File: examples_extra/ASDKgram-Swift/ASDKgram-Swift/Webservice.swift [hunk 2]
```diff
@@ -25,14 +25,15 @@ final class WebService {
...
 			let result = self.checkForNetworkErrors(data, response, error)
-			
-			switch result {
-			case .success(let data):
-				completion(resource.parse(data))
-			case .failure(let error):
-				completion(.failure(error))
-			}
-		}.resume()
+				DispatchQueue.main.async {
```
Comment (nguyenhuy) [inline examples_extra/ASDKgram-Swift/ASDKgram-Swift/Webservice.swift]: The indentation seems to be 1 tab too far?

File: examples_extra/ASDKgram-Swift/ASDKgram-Swift/Webservice.swift [hunk 3]
```diff
@@ -25,14 +25,15 @@ final class WebService {
...
+					switch result {
+					case .success(let data):
+						completion(resource.parse(data))
+					case .failure(let error):
+						completion(.failure(error))
+					}
+				}
+		}
```
Comment (nguyenhuy) [inline examples_extra/ASDKgram-Swift/ASDKgram-Swift/Webservice.swift]: Shouldn't we call `resume()` here?

Comment (onato) [reply examples_extra/ASDKgram-Swift/ASDKgram-Swift/Webservice.swift]: Indeed! Changed and retested.

**Pull request merged**